app/src/main/java/org/listenbrainz/android/viewmodel/BrainzPlayerViewModel.kt

```diff
@@ -172,6 +179,7 @@ class BrainzPlayerViewModel @Inject constructor(
             viewModelScope.launch { songRepository.updateSong(mediaItem) }
             brainzPlayerServiceConnection.transportControls.playFromMediaId(mediaItem.mediaID.toString(), null)
         }
+        _currentlyPlayingTitle.value = mediaItem.title
```

Comment:
no need to emit here, service connection should automatically update the above flow. If not, it would be a bug for BP Service connection that would need to be resolved.